pkg/acl/cache.go

```diff
@@ -148,29 +125,18 @@ func (c *cache) put(id uint32, r *Rule) {
 		return
 	}
 
-	sort.Sort(&c.id2Rule[id].NamedUsers)
```

Comment:
sorting is needed to ensure equality comparison, such as with IsEqual or calculating a checksum.

acls come from dumped json or kernel maybe unsorted.

Reply:
It's better to preserve the order from user. It's OK to have some duplicated record when the order is different

Since we use List in dumped JSON, that will be OK.